.github/workflows/mmctl-test-template.yml

```diff
@@ -34,6 +34,10 @@ jobs:
         run: |
           cd server
           make setup-go-work
+      - name: Setup needed prepackaged plugins 
+        run: |
+          cd server
+          make prepackaged-plugins PLUGIN_PACKAGES=mattermost-plugin-jira-v3.2.5
```

Comment:
Only Jira is needed in prepackaged plugins to fix: https://github.com/mattermost/mattermost/blob/e1b83360484cd84f0c19623832fef1f46727d362/server/cmd/mmctl/commands/plugin_marketplace_e2e_test.go#L20-L34

When all plugins are be included, Transitional plugins cause some catastrophic errors and mmctl tests fail.